pkg/client/unversioned/request.go

```diff
@@ -238,6 +246,35 @@ func (r *Request) AbsPath(segments ...string) *Request {
 	} else {
 		r.path = path.Join(segments...)
 	}
+	r.setAbspath = true
+	return r
+}
+
+// Group
+func (r *Request) Group(group string) *Request {
```

Comment:
Could you provide a use case of Group? When would someone set the group but not the version at the same time?